src/components/AddPoll.js

```diff
@@ -0,0 +1,99 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { handleAddPoll } from '../actions/polls';
+
+class AddPoll extends React.Component {
+  state = {
+    question: '',
+    a: '',
+    b: '',
+    c: '',
+    d: '',
+  };
+
+  handleSubmit = (e) => {
+    e.preventDefault();
+    this.props.history.push('/');
+    this.props.dispatch(handleAddPoll(this.state));
```

Comment:
maybe you wanna destruct the props here. Just a suggestion though. At the moment you having only 2 times `this.props` repeated, in bigger functions would make more sense
```suggestion
    const { history, dispatch } = this.props;
    history.push('/');    
    dispatch(handleAddPoll(this.state));
```

Reply:
Have a look if you wanna apply this change or not.